sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateClient.java

```diff
@@ -262,39 +262,16 @@ public KeyVaultCertificate updateCertificateProperties(CertificateProperties cer
      * <p>Deletes the certificate in the Azure Key Vault. Prints out the
      * deleted certificate details when a response has been received.</p>
      *
-     * {@codesnippet com.azure.security.keyvault.certificates.CertificateClient.deleteCertificate#string}
+     * {@codesnippet com.azure.security.keyvault.certificates.CertificateClient.begindeleteCertificate#string}
```

Comment:
casing - should be "beginDeleteCertificate"

Reply:
fixed.